### PR TITLE
CLI-only quickstart

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

**Comment by cremerfc** [comment]
@dexhorthy looks good to me!

File: content/vendor/guides/cli-quickstart.md [hunk 1]
```diff
@@ -45,21 +47,20 @@ $ replicated version
...
+
+#### 2. Setting an API Token
 
 Next we'll set up two environment variables. You can export these right in your shell or add them to your favorite dotfiles.
 
-```shell script
-export REPLICATED_APP=
-export REPLICATED_API_TOKEN=
-```
+If you haven't already, you'll want to head over to https://vendor.replicated.com and create an Application. You can name the app whatever you want, we'll use "CLI Quickstart" in this case. From there you can head to the settings page, grab the Application Slug, and export it:
```
Comment (marccampbell) [inline content/vendor/guides/cli-quickstart.md:56]: @codelingobot One sentence per line. Each new sentence should be written on a new line in Markdown files.

Comment (dexhorthy) [reply content/vendor/guides/cli-quickstart.md:56]: thanks, will update

Comment (Daanikus) [reply content/vendor/guides/cli-quickstart.md]: @marccampbell CodeLingo does not currently support retroactively adding check runs to existing open PRs, hence why you did not get a response from the bot. Also, please prepend capture comments with "capture".

Comment (dexhorthy) [reply content/vendor/guides/cli-quickstart.md]: @Daanikus for sure, thanks -- for now I can be the code bot 🙂

**Review by austinchambers — COMMENTED** [review]
Looks good!

**Pull request merged**